### PR TITLE
brief guide to using carpentries-theme

### PR DESCRIPTION
**Pull request by tkphd**
This PR introduces basic usage instructions. Maintainers are free to edit for clarity & correctness.

### CODE REVIEW

**Comment by tkphd** [comment]
@dpshelio, would you mind looking this over? I've probably missed some things.

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+This repository provides a remote Jekyll theme for use with The Carpentries.
+
+> !!! CAUTION !!!
+>
+> This is *early alpha* work, not yet ready for wider adoption.
+
+Many lessons built for The Carpentries carry around the website theme assets
+such as CSS and favicons. These things evolve, but keeping a given lesson
+synchronized with upstream repositories (such as [styles](
+https://github.com/carpentries/styles) and [lesson-example](
```
Comment (fmichonneau) [inline USAGE.md:12]: technically lesson example is not an upstream repository for the styling: it demonstrates how to use the styles repository

**Comment by tkphd** [comment]
Excellent points. I've called out Bundler, changed the language around Travis-CI, and delineated styles and lesson-example as separate classes of repository.

**Comment by fmichonneau** [comment]
thank you!

**Pull request merged**